packages/mjml-cli/src/client.js

```diff
@@ -83,14 +83,19 @@ export default async () => {
         type: 'object',
         describe: 'Option to pass to mjml-core',
       },
+      cp: {
```

Comment:
`--config.configPath` will be already defined, I think available options should only be listed in --config option instead